internal/pkg/util/fs/squashfs/squashfs.go

```diff
@@ -13,16 +13,15 @@ import (
 
 	"github.com/sylabs/singularity/internal/pkg/buildcfg"
 	"github.com/sylabs/singularity/pkg/runtime/engine/config"
-	singularityconfig "github.com/sylabs/singularity/pkg/runtime/engine/singularity/config"
 )
 
 // GetPath figures out where the mksquashfs binary is
 // and return an error is not available or not usable.
 func GetPath() (string, error) {
 	// Parse singularity configuration file
-	c := &singularityconfig.FileConfig{}
 	configFile := buildcfg.SINGULARITY_CONF_FILE
-	if err := config.Parser(configFile, c); err != nil {
+	c, err := config.ParseFile(configFile)
```

Comment:
`configFile` can be omitted

Reply:
no we need to parse `singularity.conf` to get path of `mksquashfs` if defined by admin otherwise we fallback and look in default paths